pinot-distribution/pinot-assembly.xml

```diff
@@ -43,10 +43,74 @@
     <file>
       <source>${pinot.root}/DISCLAIMER</source>
     </file>
+    <!-- Include Pinot All-in-one jar -->
     <file>
       <source>${pinot.root}/pinot-distribution/target/pinot-distribution-${project.version}-shaded.jar</source>
       <destName>lib/pinot-all-${project.version}-jar-with-dependencies.jar</destName>
     </file>
+    <!-- Start Include Pinot Plugins-->
+    <!-- Start Include Pinot Stream Ingestion Plugins-->
+    <!-- Only Include Specified Kafka Version Plugin-->
+    <file>
+      <source>${pinot.root}/pinot-plugins/pinot-stream-ingestion/pinot-kafka-${kafka.version}/target/pinot-kafka-${kafka.version}-${project.version}-shaded.jar</source>
```

Comment:
After I run `mvn install`, I see `pinot-plugins/target/plugins` has all shaded plugins copied. Is it possible to copy this directory at once instead of listing all files?

```
~/workspace/pinot/pinot-plugins/target/plugins adls-gen2*
❯ pwd
/Users/snlee/workspace/pinot/pinot-plugins/target/plugins

~/workspace/pinot/pinot-plugins/target/plugins adls-gen2*
❯ ll
total 0
drwxr-xr-x  11 snlee  LINKEDIN\eng   352B Mar 10 21:33 pinot-batch-ingestion
drwxr-xr-x   6 snlee  LINKEDIN\eng   192B Mar 10 21:30 pinot-file-system
drwxr-xr-x   9 snlee  LINKEDIN\eng   288B Mar 10 21:30 pinot-input-format
drwxr-xr-x   2 snlee  LINKEDIN\eng    64B Mar 10 21:29 pinot-plugins
drwxr-xr-x   6 snlee  LINKEDIN\eng   192B Mar 10 21:33 pinot-stream-ingestion
```

Reply:
That is how we did it previously by gathering all the jars together then copy directory, see line `<outputDirectory>${pinot.root}/pinot-plugins/target/plugins/${plugin.type}/${project.artifactId}</outputDirectory>` in `pinot-plugins/pom.xml`.

However `mvn release` doesn't honor this, and cannot find the built jars if we relocated them, that's why we need to hand-pick all those jars again here.